src/corespotlight.cs

```diff
@@ -979,6 +989,38 @@ interface CSSearchableItemAttributeSet : NSCopying, NSSecureCoding {
 		[Export ("valueForCustomKey:")]
 		[return: NullAllowed]
 		INSSecureCoding ValueForCustomKey (CSCustomAttributeKey key);
+
+		// CSSearchableItemAttributeSet_CSGeneral
+
+		[iOS (11,0), NoTV, Mac (10, 11)]
+		[NullAllowed, Export ("userCreated", ArgumentSemantic.Strong)]
+		NSNumber IsUserCreated { [Bind ("isUserCreated")] get; set; }
+
+		[iOS (11, 0), NoTV, Mac (10, 11)]
+		[NullAllowed, Export ("userOwned", ArgumentSemantic.Strong)]
+		NSNumber IsUserOwned { [Bind ("isUserOwned")] get; set; }
+
+		[iOS (11, 0), NoTV, Mac (10, 11)]
+		[NullAllowed, Export ("userCurated", ArgumentSemantic.Strong)]
+		NSNumber IsUserCurated { [Bind ("isUserCurated")] get; set; }
+
+		[iOS (11, 0), NoTV, Mac (10, 11)]
+		[NullAllowed, Export ("rankingHint", ArgumentSemantic.Strong)]
+		NSNumber RankingHint { get; set; }
+
+		// CSSearchableItemAttributeSet_CSItemProvider
+
+		[NoTV, Mac (10, 11)]
```

Comment:
Correct, missing the iOS decorator, fixing that. But on Watch.. CoreSpotlight is not a supported framework, and AFAIK we should not even be adding anything for the watch, as per ls on the Xcode9 contents folder:

ls /Applications/Xcode82.app/Contents/Developer/Platforms/WatchOS.platform/Developer/SDKs/WatchOS.sdk/System/Library/Frameworks
AVFoundation.framework		CoreGraphics.framework		HomeKit.framework		UIKit.framework
CFNetwork.framework		CoreLocation.framework		ImageIO.framework		UserNotifications.framework
ClockKit.framework		CoreMotion.framework		MapKit.framework		WatchConnectivity.framework
CloudKit.framework		CoreText.framework		MobileCoreServices.framework	WatchKit.framework
Contacts.framework		EventKit.framework		PassKit.framework		module.map
CoreAudio.framework		Foundation.framework		SceneKit.framework
CoreData.framework		GameKit.framework		Security.framework
CoreFoundation.framework	HealthKit.framework		SpriteKit.framework